src/components/forms/checkbox.jsx

```diff
@@ -15,9 +15,7 @@ var Checkbox = React.createClass({
             this.props.className
         );
         return (
-            <div className={classes}>
-                <FRCCheckbox {... this.props} />
-            </div>
+            <FRCCheckbox elementWrapperClassName={classes} {... this.props} />
```

Comment:
I don't agree that we want `.form-group > .[element]` (I think it's better to think `.form-group > .[component]`) since that's inverted in terms of scoping. It makes a particular component's `.form-group` inaccessible without affecting _all_ `.form-group`s. How would you put a border around one checkbox's row, but not all of them on the page?

I agree with getting rid of the `<div></div>` wrapper, but I think you want the `className` to become the `rowClassName` not the `elementWrapperClassName`.